### ISSUE TITLE
Vagrant image does not have /mnt path

### ISSUE THREAD

**Issue by JeffPsycle** [AlmaLinux/cloud-images#72]
When using the vagrant-vbguest plugin I have to ssh into the machine before it completes package installs and create the `/mnt` folder for the machine to come up cleanly.

```
Copy iso file /usr/share/virtualbox/VBoxGuestAdditions.iso into the box /tmp/VBoxGuestAdditions.iso
Mounting Virtualbox Guest Additions ISO to: /mnt
mount: /mnt: mount point does not exist.
Unmounting Virtualbox Guest Additions ISO from: /mnt
umount: /mnt: no mount point specified.
==> mymachinename: Checking for guest additions in VM...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

umount /mnt

Stdout from the command:



Stderr from the command:

umount: /mnt: no mount point specified.
```

**Comment by yippeykeiyay** [comment]
What host OS are you running?

I ran into the same error after upgrading to macOS 12.1 and fixed it by reinstalling the `vagrant-vbguest` plugin:
`vagrant plugin install vagrant-vbguest`

**Comment by LKHN** [comment]
@JeffPsycle In addition to @yippeykeiyay's comment, We need the details below to reproduce the issue.

- OS version.
- Ruby version `ruby --version`
- Installation method(Upstream or Distro provided) of the vagrant and its version. `vagrant version`
- Versions of the all installed vagrant plugins. `vagrant plugin list`

**Comment by JeffPsycle** [comment]
Thanks for the reply, here are the details:

OS Version: Fedora 34
Ruby Version: ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux]
Installation Method: Upstream provided (RPM), vagrant-2.2.19-1.x86_64

```
vagrant.x86_64                       2.2.19-1                            @hashicorp
```

Versions of all the installed plugins:

```
vagrant-cachier (1.2.1, global)
vagrant-hostmanager (1.8.9, global)
vagrant-managed-servers (0.8.0, global)
vagrant-vbguest (0.30.0, global)
```

**Comment by LKHN** [comment]
Dear @JeffPsycle  and @yippeykeiyay ,

Finally, I got time to install the VirtualBox provider and this plugin to try to produce the issue, and I did.
It's fixed on today's [almalinux/8](https://app.vagrantup.com/almalinux/boxes/8/versions/8.6.20220715) and [almalinux/9](https://app.vagrantup.com/almalinux/boxes/9/versions/9.0.20220715) releases. Please let me know if everything works for you too.

**Comment by JeffPsycle** [comment]
@LKHN thanks for the update. Unfortunately I am currently running Fedora 36 and it has a fair number of issues with VirtualBox at the moment. Waiting on VB 3.1.35 so I can test this correctly. I can confirm however that the /mnt folder is available and should get the members of my team running this on non-M1 Macs working smoothly. 

Will chime in with a status update as soon as Virtual Box updates.

**Comment by JeffPsycle** [comment]
@LKHN VirtualBox 3.1.36 just came out which has fixed the 5.18 kernel issue so I have been able to test this and confirm that the `/mnt` path issue that breaks the Guest Additions has been fixed. Thanks for getting this corrected!
